spirv_msl.cpp

```diff
@@ -103,6 +103,14 @@ bool CompilerMSL::is_msl_resource_binding_used(ExecutionModel model, uint32_t de
 	return itr != end(resource_bindings) && itr->second.second;
 }
 
+uint32_t CompilerMSL::get_resource_array_size(uint32_t id) const
+{
+	StageSetBinding tuple = { get_entry_point().model, get_decoration(id, DecorationDescriptorSet),
+		                      get_decoration(id, DecorationBinding) };
```

Comment:
Formatting.

Reply:
Hmmm...this is what `clang-format -style=file -i spirv_msl.cpp` produces on my Mac.